db/interface.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding:utf-8 -*-
+
+from db.models import Payment, Detail
+
+
+class PaymentService(object):
+    """
+    Serves as an interface which takes requests from the front-end
+    and runs the corresponding actions in the database.
+    """
+
+    def __init__(self):
```

Comment:
correct me if I'm wrong, but if we have our db = SQLAlchemy(app) in the db module init.py, does that not create the connection we need for PaymentService? we'd just need need to from db import db, right?

when it comes to python project structure and proper module usage, i have to admit some ignorance. so if i'm off base here let me know.